### ISSUE TITLE
[Bug] [BABEL] Note: The code generator has deoptimised the styling

### ISSUE THREAD

**Issue by hymair** [JuniorTour/vue-template-babel-compiler#37]
``` js
ERROR  [BABEL] Note: The code generator has deoptimised the styling of <PROJECT-PATH>/VueTemplateBabelCompiler as it exceeds the max of 500KB.
```

## Info
Only used optional chaining. Exactly 23 occurrences total in 4 different files. 

## Extra
- Usage: Nuxt.js v2.15.8
- Node.js Version: v14.19.0
- Babel Version: 
``` bash
λ npm ls --depth 0 | grep babel
+-- @babel/eslint-parser@7.17.0
+-- vue-template-babel-compiler@1.1.3```

**Comment by hymair** [comment]
Adding this did **not** fix the error.
``` js
// nuxt.config.ts
build: {
  babel: {
    compact: true,
  },
  loaders: {
    vue: {
      compiler: require('vue-template-babel-compiler'),
    },
  },
}
```

**Comment by JuniorTour** [comment]
🤔 This msg seems to be just a `warning` from babel?

Refer: https://stackoverflow.com/questions/35192796/babel-note-the-code-generator-has-deoptimised-the-styling-of-app-js-as-it-exc

<hr />

Does it have any negative impact for you?

If you don't want to see this warning msg, you can add `compact: true` to babel options: https://github.com/JuniorTour/vue-template-babel-compiler/blob/main/doc/Usage.md#1-babel-options-customization

**Comment by hymair** [comment]
`compact: true` hides the error but I had to add `@ts-ignore` as there was a type error.

```
Type '{ babelOptions: { compact: true; }; }' is not assignable to type 'CompilerOptions'.
  Object literal may only specify known properties, and 'babelOptions' does not exist in type 'CompilerOptions'.
```
